### PR TITLE
GAMBLR.data cleanup

### PR DESCRIPTION
**Pull request by mattssca**
This PR adds (or rather removes) a few outstanding things that didn't get into the previous PR before it was merged. These things are:

- Removing traces of lingering Grande MAF
- Adding GAMBLR.helpers under imports in DESCRIPTION
- Adding a new line at e end of the per script (inst/perl/check_functions.pl)

### CODE REVIEW

File: data-raw/DATASET.R [hunk 1]
```diff
@@ -365,14 +365,6 @@ lymphoma_genes = dplyr::select(lymphoma_genes,-entrezgene_id) %>% group_by(Gene,
 #lymphoma_genes_comprehensive = dplyr::filter(lymphoma_genes_comprehensive,Reddy==TRUE | Chapuy == TRUE | LymphGen == TRUE | curated == TRUE)
 #lymphoma_genes_comprehensive$other_support = ""
 
-# Example SSM data from Grande et. al, 2019
-grande_maf = system.file("extdata", "blood8871418-suppl2-ssm.csv", package = "GAMBLR.data") %>%
-  read_tsv(.) %>%
-  as.data.frame
-grande_maf = grande_maf[!colnames(grande_maf) %in% c("tumor_biospecimen_id","normal_biospecimen_id")]
-
-usethis::use_data(grande_maf, overwrite = TRUE)
```
Comment (vladimirsouza) [inline data-raw/DATASET.R:374]: I might miss that, but why `grande_maf` was deleted?

Comment (mattssca) [reply data-raw/DATASET.R:374]: With the introduction of the new dataset, called [sample_data](https://github.com/morinlab/GAMBLR.data/blob/master/man/sample_data.Rd), the Grande MAF is redundant in many ways. This new dataset has metadata as well as maf, seg, and bedpe data for both projections bundled in the same dataset.

File: DESCRIPTION [hunk 2]
```diff
@@ -26,6 +26,7 @@ Depends:
 LazyData: true
 Imports: 
     dplyr,
+    GAMBLR.helpers,
```
Comment (vladimirsouza) [inline DESCRIPTION]: Do we really need `GAMBLR.helpers` here? I think there is no `@import GAMBLR.helpers` tag in any function documentation.

File: data-raw/DATASET.R (hunk 1 above)
Comment (vladimirsouza) [reply data-raw/DATASET.R:374]: Understood. Thanks for the explanation.

File: DESCRIPTION (hunk 2 above)
Comment (mattssca) [reply DESCRIPTION]: Let me try if it can be omitted from the DESCRIPTION.

File: data-raw/DATASET.R
Comment (vladimirsouza) [inline data-raw/DATASET.R:1]: In the line
```
reddy_genes = system.file("extdata","reddy_genes.tsv",package="GAMBLR") %>%
```
shouldn't it be `GAMBLR.data`?

Comment (mattssca) [reply data-raw/DATASET.R:1]: Good find, yes it should be `GAMBLR.data`. I thought I caught them all. Thanks!

**Review by vladimirsouza — APPROVED** [review]

**Pull request merged**